### PR TITLE
add code and readme for Event-Driven Network Automation with NetBox and Ansible Automation Platform webinar

### PR DESCRIPTION
**Pull request by richbibby**

### CODE REVIEW

File: netbox-event-driven-ansible/README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+This joint session with NetBox Labs and Red Hat will showcase the seamless integration of NetBox as a Source of Truth with Ansible Automation Platform (AAP) and Event-Driven Ansible (EDA).
+We’ll delve into NetBox’s latest features, including Branching and Change Management, and highlight new capabilities in AAP 2.5, such as the Unified UI with EDA and the interactive walkthroughs.
+
+The live demo will feature dynamic inventory updates in AAP powered by NetBox, illustrating how event-driven automation workflows in EDA are triggered by NetBox events and data.
+
+Use cases will include updating NTP server and VLAN configurations, as well as provisioning new devices demonstrating a streamlined, automated approach to network and infrastructure management.
+
+All registrants will receive the recording and slides after the live webinar
+
+## Read More in the Blog
```
Comment (mrmrcoleman) [inline netbox-event-driven-ansible/README.md:14]: Perhaps link to the actual blog

File: netbox-event-driven-ansible/README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+You can also read an in-depth overview of this solution on the NetBox Labs [Blog](https://netboxlabs.com/blog/)
+
+## Code Overview
+
+The code is split over two directories:
+
+### playbooks
+These are the playbooks that push configuration changes out to the network devices. These are run as jobs by the Automation Execution element of Ansible Automation Platform.
```
Comment (mrmrcoleman) [inline netbox-event-driven-ansible/README.md]: I think it should be "They are run as..."

File: netbox-event-driven-ansible/README.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### rulebooks
+This code defines the rules that trigger jobs in Event Driven Ansible and configures the port that EDA listens on for NetBox event data arriving in the form of webhooks.
+
+## Demo Environment
+The demo environment consists of:
```
Comment (mrmrcoleman) [inline netbox-event-driven-ansible/README.md:28]: Can readers run this themselves?

File: netbox-event-driven-ansible/rulebooks/aws-sqs-rulebook-template.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+
+- name: Events on SQS
+  sources:
+    - ansible.eda.aws_sqs_queue:
+        access_key: "{{ aws_access_key }}"
```
Comment (mrmrcoleman) [inline netbox-event-driven-ansible/rulebooks/aws-sqs-rulebook-template.yml]: Is it using Event Streams or webhooks?

File: netbox-event-driven-ansible/README.md (hunk 1 above)
Comment (richbibby) [reply netbox-event-driven-ansible/README.md:14]: yep will update this once the blog is live and I have a URL

File: netbox-event-driven-ansible/README.md (hunk 2 above)
Comment (richbibby) [reply netbox-event-driven-ansible/README.md]: fixed

File: netbox-event-driven-ansible/rulebooks/aws-sqs-rulebook-template.yml (hunk 4 above)
Comment (richbibby) [reply netbox-event-driven-ansible/rulebooks/aws-sqs-rulebook-template.yml]: good point this is placeholder code for SQS, I will remove it

**Review by mrmrcoleman — APPROVED** [review]

**Pull request merged**